Mathlib/Algebra/BigOperators/Group/List.lean

```diff
@@ -432,9 +432,9 @@ lemma prod_rotate_eq_one_of_prod_eq_one :
   | [], _, _ => by simp
   | a :: l, hl, n => by
     have : n % List.length (a :: l) ≤ List.length (a :: l) := le_of_lt (Nat.mod_lt _ (by simp))
-    rw [← List.take_append_drop (n % List.length (a :: l)) (a :: l)] at hl;
```

Comment:
And I thought I had caught all such lines at one point - thanks for fixing this!